server/routes/api/index.js

```diff
@@ -1,9 +1,13 @@
 import express from 'express';
+import swaggerUi from 'swagger-ui-express';
+import swaggerSpec from '../../config/swagger';
 import userRoutes from './users';
 
 const apiRoutes = express.Router();
 
 apiRoutes.use('/', userRoutes);
 
+// swagger-ui-express for API endpoint documentation
+apiRoutes.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerSpec));
```

Comment:
I believe this should be in the `app.js` file then?

Reply:
@fidelisojeah fixed. Thanks